src/Schema/Keywords/Type.php

```diff
@@ -39,7 +40,7 @@ public function validate($data, string $type, ?string $format = null) : void
     {
         switch ($type) {
             case CebeType::BOOLEAN:
-                if (! is_bool($data)) {
+                if (! is_bool($data) && ! preg_match('#^(true|false)$#i', (string) $data)) {
```

Comment:
string cast on unknown type of $data may cause a notice: "Notice: Array to string conversion"

Reply:
Wow, I wouldn't have thought about that!